package.json

```diff
@@ -19,5 +19,9 @@
   "bugs": {
     "url": "https://github.com/concrete-utopia/utopia/issues"
   },
-  "homepage": "https://utopia.app"
+  "homepage": "https://utopia.app",
+  "dependencies": {
+    "@radix-ui/themes": "2.0.3",
```

Comment:
I think these are misplaced, you probably accidentally ran `pnpm install @radix-ui/themes` in the root directory instead of the utopia-remix folder. we should revert this and the package-lock change, and re-run `pnpm install ...` inside `utopia-remix`.

I can help if you need it :)